examples/UAVCAN-BMS-Subscribe/UAVCAN-BMS-Subscribe.ino

```diff
@@ -0,0 +1,207 @@
+/*
+ *  This example file shows how to read messages from a UAVCAN v1 enable BMS and display
+ *   the information in console output.
+ *
+ *  Hardware:
+     - UAVCAN v1 enabled BMS (eg: https://rotoye.com/batmon/ and
+        wiring guide: https://rotoye.com/batmon-arduino-can-bus-wiring-guide-and-uavcan-v1/)
+     - A (multicell) battery or battery simulator
+ *   - Arduino MKR family board, e.g. MKR Zero
+ *   - Arduino MKR CAN shield
+ *
+ *  Copyright (c) 2020, SkyMul Inc.
+ *  All Rights Reserved. Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *  * Adheres to the rules of GNU GENERAL PUBLIC LICENSE, Version 3
+ *  * Redistributions of source code must retain this above copyright notice, this
+ *      list of conditions and the following disclaimer.
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *      this list of conditions and the following disclaimer in the documentation
+ *      and/or other materials provided with the distribution.
+ *  * Neither the name of the copyright holder nor the names of its
+ *      contributors may be used to endorse or promote products derived from
+ *      this software without specific prior written permission.
+ *  Author: Kyle Pawlowski
+ *          Eohan George eohan@rotoye.com
+ */
+
+/**************************************************************************************
+ * INCLUDE
+ **************************************************************************************/
+
+#include <SPI.h>
+
+#include <ArduinoUAVCAN.h>
+#include <ArduinoMCP2515.h>
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
+ using namespace reg::drone::service::battery;
+ using namespace reg::drone::physics::electricity;
+
+/**************************************************************************************
+ * CONSTANTS
+ **************************************************************************************/
+
+static int const MKRCAN_MCP2515_CS_PIN  = 3;
+static int const MKRCAN_MCP2515_INT_PIN = 7;
+static CanardPortID const SOURCETS_PORT_ID   = 6144U;
+static CanardPortID const STATUS_PORT_ID   = 6145U;
+static CanardPortID const PARAMETERS_PORT_ID   = 6146U;
```

Comment:
This bit violates section **2.1.2.2 Regulation** of the UAVCAN Specification. It can be fixed by adding two RPC-services: uavcan.register.List and uavcan.register.Access, kind of like it's done in this old demo I wrote some time ago: [nonfixed_port_id.tar.gz](https://github.com/107-systems/107-Arduino-UAVCAN/files/5974934/nonfixed_port_id.tar.gz). Would you consider fixing it?

Reply:
Good morning @pavel-kirienko :wave: In this case the [UAVCAN-Blink](https://github.com/107-systems/107-Arduino-UAVCAN/blob/master/examples/UAVCAN-Blink/UAVCAN-Blink.ino) example would also violate that part of the specification. I suggest it would be best to think of the examples within this repository as "MVPs", that is the smallest possible codebase to get something (anything) done with UAVCAN. However, I'd love to introduce those RPC-Services you just described to the WIP reference implementations ([UAVCAN-GNSS-Node](https://github.com/107-systems/UAVCAN-GNSS-Node) and [UAVCAN-ToF-Distance-Sensor-Node](https://github.com/107-systems/UAVCAN-ToF-Distance-Sensor-Node)). Would that be allright with you?

Reply:
>In this case the UAVCAN-Blink example would also violate that part of the specification.

Yes. I guess we can keep things as-is but can we please make it clear in the affected examples that they are not fully spec-compliant? The thought of "UAVCAN-enabled" products that violate the Regulation requirements appearing on the market keeps me up at night.

Reply:
Well, I wouldn't want to rob you of your sleep - would #95 accommodate you?